config/exporters/monitoring/prometheus/base/prometheus-exporter-service.yaml

```
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: exporter-sa
+  namespace: appstudio-exporter-service-prometheus
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: exporter-cluster-role
+rules:
+- apiGroups: ['grafana.integreatly.org']
+  resources: ['grafanas']
+  verbs: ["get"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: exporter-cluster-role-binding
+subjects:
+- kind: ServiceAccount
+  name: exporter-sa
+  namespace: appstudio-exporter-service-prometheus
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: exporter-cluster-role
```

```suggestion
apiVersion: rbac.authorization.k8s.io/v1
kind: RoleBinding
metadata:
  name: exporter-cluster-role-binding
  namespace: appstudio-grafana
subjects:
- kind: ServiceAccount
  name: exporter-sa
  namespace: appstudio-exporter-service-prometheus
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: exporter-cluster-role
```

This way we restrict the access defined in the `ClusterRole` only to `appstudio-grafana` namespace

---

Ok, will update it.